source/guide/index.md

```diff
@@ -10,124 +10,125 @@ version: 2.1
 
-Within the `<template>` of the source code, You must be familiar with the `<div>` which is widely used on the web, and it's also the generic container on Weex. But the `<text>` component is provided by Weex, it's a block-level text container.
+Within the `<template>` of the source code, you will notice the `<div>` element, which is widely used on the web, and is also the generic container in Weex. The `<text>` component, however, is provided by Weex and is a block-level text container.
 
 > The raw text node can only be placed in the `<text>` component, otherwise, it will be ignored.
 
 Within the `<style>` tag, you can write CSS to describe the styles of a component, and those styles are [**scoped**](https://vue-loader.vuejs.org/en/features/scoped-css.html) forcibly in Weex.
 
 ### Native Components
 
-In the example above, the `<div>` and the `<text>` is rendered into corresponding native views on the mobile device, which is not `HTMLElement`.
+In the example above, the `<div>` and the `<text>` elements are rendered into corresponding native views on the mobile device. As such, they do not implement the `HTMLElement` interface.
 
 ![Native Components](./images/native-component.png)
 
 Weex implements render engines both on iOS and Android and provides a group of [built-in components](../references/components/) for basic usage. Based on these components, you can compose and wrap more custom components.
 
 Although the components in Weex look like HTML tags, you are not able to use all of them, instead, you can only use the built-in components and your custom components.
 
-Under the scene, Weex uses the native widgets. Although Weex emphasize the consistency on each mobile platform, we still embrace the platform's own behavior and UI differences. For example, [the `<switch>` component](http://dotwe.org/vue/d96943452b6708422197c47920903823) may look different on Android and iOS (the appearance on the web simulates iOS).
+Behind the scenes, Weex uses native widgets. Although Weex emphasizes consistency on each mobile platform, we still embrace the platform's own behavior and UI differences. For example, [the `<switch>` component](http://dotwe.org/vue/d96943452b6708422197c47920903823) may look different on Android and iOS (the appearance on the web simulates iOS).
 
 ![Different switch](./images/different-switch.png)
 
-If you want to use more native component other than the built-in components, you need to implement them on each platform and keep their behaviors to be consistent. The most practical way is to integrate the existing native components to Weex platform.
+If you want to use additional native components, other than the built-in components Weex provides, you need to implement them on each platform and keep their behaviors consistent. The most practical way is to integrate the existing native components to Weex platform. /_ need explanation _/
```

Comment:
We can change "...the built-in components Weex provides,..." to "...the built-in components provided by Weex,..."